doc/source/operations/tempest.rst

```diff
@@ -0,0 +1,303 @@
+======================================
+Running Tempest with Kayobe Automation
+======================================
+
+Overview
+========
+
+This document describes how to configure and run `Tempest
+<https://docs.openstack.org/tempest/latest/>`_ using `kayobe-automation
+<https://github.com/stackhpc/kayobe-automation>`_ from the ``.automation``
+submodule included with ``stackhpc-kayobe-config``.
+
+The following guide will assume all commands are run from your
+``kayobe-config`` root and the environment has been configured to run Kayobe
+commands unless stated otherwise.
+
+Building a Kayobe container
+===========================
+``kayobe-automation`` runs in a container on the Ansible control host. This
+means that Docker must be installed on the Ansible control host if it is not
+already.
+
+.. warning::
+
+    Docker can cause networking issues when it is installed. By default it will
+    create a bridge and change ``iptables`` rules. These can be
+    disabled by setting the following in ``/etc/docker/daemon.json``:
+
+    .. code-block:: json
+
+        {
+            "bridge": "none",
+            "iptables": false
```

Comment:
Worth noting that this breaks any GitHub Actions runners that may be sharing the host, right @jackhodgkiss?

Reply:
Actually are we missing this one still?

Reply:
If `iptables` are disabled then and GitHub runner will fail to access the internet as host networking is not permitted for container workloads.

See: https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container#setting-container-resource-options

Also worth mentioning that in all the time I have been using kayobe-automation I have never needed to disable ip-tables. I understand some have reported issues Docker but I cannot say the same.

Reply:
Thanks Jack, I'll update the PR soon to give a better description

Reply:
Done